src/components/Home.jsx

```diff
@@ -19,24 +22,71 @@ function Home() {
     const token = getToken();
     localStorage.setItem('list-token', token);
 
-    //add token to Firestore
+    //add token to Firestore under 'token' collection
     try {
       const docRef = await addDoc(collection(db, 'token'), {
         token,
       });
+
       navigate('/listView');
       console.log('Document written with ID: ', docRef.id);
     } catch (e) {
       console.error('Error adding document: ', e);
     }
   };
 
+  const handleTokenSubmit = async () => {
+    //create query for token collection in firestore
+    const q = query(collection(db, 'token'), where('token', '==', userToken));
+
+    //set token in local storage if it exists
+    const querySnapshot = await getDocs(q);
+    querySnapshot.forEach((doc) => {
+      localStorage.setItem('list-token', doc.data().token);
+    });
+
+    const localToken = localStorage.getItem('list-token');
+    if (localToken) {
+      navigate('/listView');
+    } else {
+      setNotification(
+        'The token does not exist. try again or create a new list',
+      );
+      setUserToken('');
+      //setTimeout is used to clear the notification after 2 seconds
+      setTimeout(() => {
+        setNotification('');
+      }, 3000);
+      console.log(`'${userToken}' token does not exist...`);
+    }
+  };
+
   return (
     <div className="App">
-      <header className="App-header">
+      <main className="App-header">
         <p>Our Shopping App</p>
-        <button onClick={() => handleNewList()}>Create New List</button>
-      </header>
+        <button onClick={() => handleNewList()}>Create a New List</button>
+        <div>
+          <p>--Or--</p>
+          <p>Join an Existing List by Entering Three Word Token</p>
+        </div>
+        <label htmlFor="userToken">List Token:</label>
+        <input
+          type="text"
+          id="userToken"
+          name="userToken"
+          value={userToken}
+          onChange={(e) => setUserToken(e.target.value)}
```

Comment:
we can add here e.targer.value.toLowerCase()
What do you think @djtaylor8 
or we can do the same on line #40

Reply:
Either of those would be good. We could also create a separate function to check for extra characters (numbers, punctuation, etc). That would take care of it in one place. But for now, going to lowercase works well...